### PR TITLE
Add recipe for mmoreram/gearman-bundle 4.1+

### PR DESCRIPTION
**Pull request by pinkeen**
| Q             | A
| ------------- | ---
| License       | MIT

Adds recipe for gearman-bundle. Adding doctrine cache provider this way is safe because this config key is merged.

### CODE REVIEW

**Comment by daum** [comment]
@Nyholm this should be good now.

File: mmoreram/gearman-bundle/4.1/config/packages/gearman.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    providers:
+        gearman_cache:
+            type: file_system
+            namespace: doctrine_cache.ns.gearman
+
+# For full configuration reference please see:
+# http://gearmanbundle.readthedocs.io/en/latest/configuration.html
+#
+# At least you need to provide `resources` or `bundles` configuration in order for your
+# worker annotations to be disoverable.
```
Comment (fabpot) [inline mmoreram/gearman-bundle/4.1/config/packages/gearman.yaml]: typo: `discoverable`

**Comment by daum** [comment]
@pinkeen Thanks again for this.  I'll update the bundles readme to state a recipe exists, sound good?

**Comment by pinkeen** [comment]
Sure, sounds good

**Pull request merged**